src/ScottPlot5/ScottPlot5/Primitives/SvgImage.cs

```diff
@@ -6,28 +6,42 @@ internal class SvgImage : IDisposable
     public readonly int Width;
     public readonly int Height;
     private readonly MemoryStream Stream;
-    public readonly SKCanvas Canvas;
+    public SKCanvas Canvas
```

Comment:
This is cool, I guess I'll let Scott comment on whether we need to support reusing the `SvgImage` object or if this is too fancy.